system/jlib/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ set (    SRCS
          jexcept.cpp 
          jfile.cpp 
          jflz.cpp 
+         jlz4.cpp
+         jden.cpp
```

Comment:
Ok, I see why you've done like this, so that the DEN compressor/expander methods are implemented to throw exceptions to report unsupported..
That does seem reasonable.